scripts/base_network.py

```diff
@@ -38,14 +38,14 @@
 ------
 
 - ``data/entsoegridkit``:  Extract from the geographical vector data of the online
-  `ENTSO-E Interactive Map <https://www.entsoe.eu/data/map/>`_ by the
-  `GridKit <https://github.com/pypsa/gridkit>`_ toolkit dating back to January 2020.
+`ENTSO-E Interactive Map <https://www.entsoe.eu/data/map/>`_ by the
+`GridKit <https://github.com/pypsa/gridkit>`_ toolkit dating back to January 2020.
 - ``data/parameter_corrections.yaml``: Corrections for ``data/entsoegridkit``
 - ``data/links_p_nom.csv``: confer :ref:`links`
 - ``data/links_tyndp.csv``: List of projects in the
-  `TYNDP 2018 <https://tyndp.entsoe.eu/tyndp2018/>`_ that are at least *in permitting*
-  with fields for start- and endpoint (names and coordinates), length, capacity,
-  construction status, and project reference ID.
+`TYNDP 2018 <https://tyndp.entsoe.eu/tyndp2018/>`_ that are at least *in permitting*
+with fields for start- and endpoint (names and coordinates), length, capacity,
+construction status, and project reference ID.
 - ``resources/country_shapes.geojson``: confer :ref:`shapes`
```

Comment:
Similarly to above, please, set the docstring here as in new-main

Reply:
It's updated, no?